qa/automation/CareerPortal.Tests.Scenarios/Savostin/CV_sending.feature

```diff
@@ -0,0 +1,179 @@
+	When user clicks on "Send" button
+	And all fields are empty except name with <name>
+	Then form should not be sent
+	And following errors should be displayed appropriate fields:
+	|"this field is required"                             |
+	|"attach CV" becomes red                              |
+	|"I agree to the personal data processing" becomes red|
+
+	Example: Eugene Savostin
+
+Scenario Outline:1.0.0_04 form with only invalid email on it can not be sent
+	When user clicks on "Send" button
+	And all fields are empty except email with <email>
+	Then form should not be sent
+	And following errors should be displayed appropriate fields:
+	|"this field is required"                             |
+	|"invalid email format"                               |
+	|"attach CV" becomes red                              |
+	|"I agree to the personal data processing" becomes red|
+
+	Examples: 
+	|email      |
+	|dadassdf   |
+	|dadada@mail|
+
+Scenario Outline:1.0.0_05 form with only valid email on it can not be sent
+	When user clicks on "Send" button
+	And all fields are empty except email with <email>
+	Then form should not be sent
+	And following errors should be displayed appropriate fields:
+	|"this field is required"                             |
+	|"attach CV" becomes red                              |
+	|"I agree to the personal data processing" becomes red|
+
+	Examples: 
+	|email            |
+	|myemail@gmail.com|
+	|test@mail.ru     |
+
+Scenario:1.0.0_06 form with only attached CV on it can not be sent
+	When user clicks on "Send" button
+	And all fields are empty except CV "valid_file_to_attach.docx" is attached
+	Then form should not be sent
+	And following errors should be displayed appropriate fields:
+	|"this field is required"|
+	|"this field is required"|
+	|"I agree to the personal data processing" becomes red|
+
+Scenario Outline:1.0.0_07 form with all valid data except name on it can not be sent
+	Given field name contains <name>
+	And field email contains <email>
+	And CV "Valid_file_to_attach.docx" is attached
+	And confirmation of data processing is true
+	When user clicks on "Send" button
+	Then form should not be sent
+	And following error should be displayed under field "name":
+	|"invalid name format"|
+
+	Examples: 
+	|name  |email           |
+	|Eugene|eugene@gmail.com|
+
+Scenario Outline:1.0.0_08 form with all valid data except email on it can not be sent
+	Given field name contains <name>
+	And field email contains <email>
+	And CV "Valid_file_to_attach.docx" is attached
+	And personal data processing is confirmed
+	When user clicks on "Send" button
+	Then form should not be sent
+	And following error should be displayed under field "email":
+	|"invalid email format"|
+
+	Examples: 
+	|name           |email              |
+	|Marina Sidorova|thisMailIsIncorrect|
+
+Scenario:1.0.0_09 form can not take file with invalid type
+	When user chooses file with invalid type to input
```

Comment:
All test data should be prepared and used in your test cases.
We can't say - some files should be uploaded. No, only specific files should be uploaded.

This comment is valid for all similar mistakes through whole feature file.